src/app/lib/utilities/srcSet/index.js

```diff
@@ -2,39 +2,63 @@ import buildIChefUrl from '#lib/utilities/ichefURL';
 
 const DEFAULT_RESOLUTIONS = [240, 320, 480, 624, 800];
 
-export const createSrcset = (
+export const createSrcsets = ({
   originCode,
   locator,
   originalImageWidth,
-  resolutions = DEFAULT_RESOLUTIONS,
-) => {
+  imageResolutions = DEFAULT_RESOLUTIONS,
+}) => {
   if (originCode === 'pips') {
-    return null;
+    return { webpSrcset: null, fallbackSrcset: null };
   }
 
-  const requiredResolutions = resolutions.filter(
+  const requiredResolutions = imageResolutions.filter(
     resolution => resolution <= originalImageWidth,
   );
 
   if (
-    originalImageWidth < resolutions[resolutions.length - 1] &&
+    originalImageWidth < imageResolutions[imageResolutions.length - 1] &&
     !requiredResolutions.includes(originalImageWidth)
   ) {
     requiredResolutions.push(originalImageWidth);
   }
 
-  const urls = requiredResolutions.map(
-    resolution =>
-      `${buildIChefUrl({ originCode, locator, resolution })} ${resolution}w`,
-  );
+  const webpSrcset = requiredResolutions
+    .map(
+      resolution =>
+        `${buildIChefUrl({
+          originCode,
+          locator,
+          resolution,
+          isWebP: true,
+        })} ${resolution}w`,
+    )
+    .join(', ');
+
+  const fallbackSrcset = requiredResolutions
+    .map(
+      resolution =>
+        `${buildIChefUrl({
+          originCode,
+          locator,
+          resolution,
+          isWebP: false,
+        })} ${resolution}w`,
+    )
+    .join(', ');
```

Comment:
Could maybe be shortened to:

```suggestion
    const [webpSrcset, fallbackSrcset] = [true, false].map(isWebP =>
    requiredResolutions
        .map(
        resolution =>
            `${buildIChefUrl({
            originCode,
            locator,
            resolution,
            isWebP,
            })} ${resolution}w`,
        )
        .join(', '),
    )

```

I haven't tested this code though

Reply:
Nice, works well 👍🏼